### PR TITLE
Do not show "ENS" label in Actionsheet for transaction confirmation under Recipient if there is no ENS name resolved (yet) #2310

### LINKED ISSUE #2310: Do not show "ENS" label in Actionsheet for transaction confirmation under Recipient if there is no ENS name resolved (yet)
If we tap on Recipient in the actionsheet, we see (A):

After a while, it becomes (B):

Let's make it be (B) unless an ENS name has already been resolved (or provided) and become (A) if/when it's resolved.

### PR DESCRIPTION
**Pull request by vladyslav-iosdev**
Closes #2310

### CODE REVIEW

File: AlphaWallet/Transfer/ViewControllers/TransactionConfirmationViewController.swift [hunk 1]
```diff
@@ -453,14 +454,14 @@ extension TransactionConfirmationViewController {
             for (sectionIndex, section) in viewModel.sections.enumerated() {
                 let header = TransactionConfirmationHeaderView(viewModel: viewModel.headerViewModel(section: sectionIndex))
                 header.delegate = self
-                var children: [UIView] = []
+//                var children: [UIView] = []
```
Comment (hboon) [inline AlphaWallet/Transfer/ViewControllers/TransactionConfirmationViewController.swift:457]: Can you remove the commented out code in the commit?

File: AlphaWallet/Transfer/ViewControllers/TransactionConfirmationViewController.swift [hunk 2]
```diff
@@ -453,14 +454,14 @@ extension TransactionConfirmationViewController {
...
                 switch section {
                 case .gas:
                     header.setEditButton(section: sectionIndex, self, selector: #selector(editTransactionButtonTapped))
                 case .amount:
                     break
                 }
-                header.childrenStackView.addArrangedSubviews(children)
+//                header.childrenStackView.addArrangedSubviews(children)
```
Comment (hboon) [inline AlphaWallet/Transfer/ViewControllers/TransactionConfirmationViewController.swift:464]: Same here

**Review by hboon — CHANGES_REQUESTED** [review]
If we add a delay to slow down the reverse resolution in `ENSReverseLookupCoordinator` and go proceed to try to send to 0xd8da6bf26964af9d7eed9e03e53415d37aa96045 (which should resolves to `vitalik.eth` in a while), this screen is still shown for a while until the ENS name appears:

**Comment by vladyslav-iosdev** [comment]
> If we add a delay to slow down the reverse resolution in ENSReverseLookupCoordinator and go proceed to try to send to 0xd8da6bf26964af9d7eed9e03e53415d37aa96045 (which should resolves to vitalik.eth in a while), this screen is still shown for a while until the ENS name appears:

oh, i didn't check this case, working on it.

**Comment by hboon** [comment]
@vladyslav-iosdev looks good. Is it go to review and merge?

**Comment by hboon** [comment]
@vladyslav-iosdev build error. Maybe the `children` var was deleted when rebasing?

**Review by hboon — APPROVED** [review]
Thanks!

**Pull request merged**